index.html

```diff
@@ -5,105 +5,125 @@
+                                font-size: 3vw;
+                                margin-left: 0;
+                                text-align: center;
+                                width:initial;
+                            }
+                            #mainbuttons{
+                                display: flex;
+                                justify-content: center;
+                                flex-direction:row;
+                                margin-left:0;
+                                width: 100%;
+                                margin-top: 3vh;
+                            }
+
+                            #mainbuttons #button{
+                                font-size: 3vw;
+                                margin-left: 1vh;
+                                margin-right: 1vh;
+                            }
+
+                            #rightcontainer{
+                                margin-top: 20vh;
+                                height: initial;
+                                display: flex;
+                                flex-direction: column;
+                                justify-content: center;
+                                margin-right: 10vw;
+                                margin-left: 10vw;
+                                min-width: 0vw;
+                            }
+                            #right{
+                                width: 100%;
+                                margin-right: 0;
+                            }
+
+                            #otherbuttons{
+                                align-items: center;
+                                justify-content: center;
+                                width: inherit;
+                            }
+
+                            #otherbuttons #button,#otherbuttons #button:visited{
+                                font-size: 3vw;
+                                border: 0.1vh solid gainsboro;
+                                margin-left: 2vw;
+                                margin-right: 2vw;
+                            }
+                            #end{
+                                margin-bottom: 1vh;
+                                margin-right: 2vw;
+                                margin-top: 2vh;
+                                position: initial;
+                                text-align: right;
+                            }
+                            `;
+            ///////////////////////////////////css end/////////////////////////////
+            var styleSheet = document.createElement("style")
+            var w = window.innerWidth;
+            var h = window.innerHeight;
+            if (w/h < 1.45){         //if aspect ratio is small enough
+                styleSheet.type = "text/css";
+                styleSheet.innerText = styles;
+                document.head.appendChild(styleSheet);
+            }
+
+        
+        //////////////////////Javscript end/////////////////////////////
+        </script>
```

Comment:
Can you please move the `script`-part to `web/js/` and include it from there, so we have everything at one place.

Reply:
Can this be done with a CSS media query instead? 
The [`aspect-ratio`](https://developer.mozilla.org/en-US/docs/Web/CSS/@media/aspect-ratio) query is supported pretty universally.

```css
@media (max-aspect-ratio: 3/2) {
  /* put the css here */
}
```

Reply:
I forgot that existed, just finished the fixes